client/test/integration/index.ts

```diff
@@ -39,6 +39,7 @@ export async function run(): Promise<void> {
     // mocha.reporter('xunit', { output: resultsFileName });
 
     mocha.options.color = true;
+    mocha.timeout(15000);
```

Comment:
Why 15s? I think it's good to comment all wait/timeout values like this so a developer six months from now understands why this particular value was chosen and not e.g. 10s or 20s.